libraries/apollo-debug-server/build.gradle.kts

```diff
@@ -123,3 +126,11 @@ dependencies {
     to.attribute(artifactType, ArtifactTypeDefinition.DIRECTORY_TYPE)
   }
 }
+
+// Can't use apiVersion KOTLIN_2_0 when using languageVersion KOTLIN_1_9, which is the case here because we're using KSP 1
```

Comment:
Where is `languageVersion` set to `KOTLIN_1_9`?  Is KSP overriding our default?

Reply:
I'm not actually sure, but guessing yes, KSP must automatically switch to K1. Interestingly, if I set  `languageVersion.set(KotlinVersion.KOTLIN_1_9)` here, I'm getting an error:
```
Inconsistent settings for Kotlin source sets: 'jvmMain' depends on 'jvmCommonMain'
  'jvmMain': language version is KOTLIN_1_9
  'jvmCommonMain': language version is KOTLIN_2_0
  The language version of the dependent source set must be greater than or equal to that of its dependency.
```